### ISSUE TITLE
More customization for TabItem (e.g. shadow, elevation)

### ISSUE THREAD

**Issue by cosinus84** [hacktons/convex_bottom_bar#25]
I am trying to make the middle tabItem to act more as FAB. First problem that I encountered is that the activeColor/Color is working for the whole Bar and not on each TabItem
```
ConvexAppBar(
        style: TabStyle.fixedCircle,
        color: Colors.white,
        curveSize: 0,
        items: [
          TabItem(icon: Icons.home, title: 'Home'),
          TabItem(icon: Icons.map, title: 'Discovery'),
          TabItem(icon: Icons.add, title: 'Add'),
          TabItem(icon: Icons.message, title: 'Message'),
          TabItem(icon: Icons.people, title: 'Profile'),
        ],
        onTap: (int i) => print('click index=$i'),
      )
```

Thanks and good job!

**Comment by avenwu** [comment]
Yes, these configurations are used for all tab items.
in your situation, please try with ConvexAppBar.builder

There is a example show how to use the builder constructor. Feel free to check the README and have a look at the online preview.

https://appbar.codemagic.app/#/custom

The custom example is not exactly what you've described, but you may define a builder with isolated effects on center tab item to make it as a FAB.

**Comment by cosinus84** [comment]
Hello, also if the curveSize is 0 the tap is not working on the part of the button that is above the bar.
![Screenshot 2020-02-12 at 17 23 12](https://user-images.githubusercontent.com/1459132/74350192-0585c800-4dbe-11ea-9ee9-562d7106925a.png)

Thanks

**Comment by avenwu** [comment]
do you mean the click event is not responsive on the top area of button? it'd weird, will have a check.

**Comment by avenwu** [comment]
@cosinus84 the click issue has been fixed; Update to v2.0.3 should just work
